$NetBSD: patch-rebar.config,v 1.1 2021/01/16 00:00:48 triaxx Exp $

On BSD systems there is no libdl, as the required functionality is already built
into libc.

--- rebar.config.orig	2020-12-08 13:55:42.000000000 +0000
+++ rebar.config
@@ -25,7 +25,7 @@
             {if_have_fun, {rand, uniform, 1}, {d, 'RAND_UNIFORM'}}]}.
 
 {port_env, [{"ERL_LDFLAGS", " -L$ERL_EI_LIBDIR -lei"},
-            {"CFLAGS", "$CFLAGS"}, {"LDFLAGS", "$LDFLAGS -ldl"}]}.
+            {"CFLAGS", "$CFLAGS"}, {"LDFLAGS", "$LDFLAGS"}]}.
 
 {port_specs, [{"priv/bin/eimp", ["c_src/eimp.c"]}]}.
 
